x-pack/plugins/security_solution/public/flyout/document_details/right/components/risk_score.tsx

```diff
@@ -25,9 +26,9 @@ export const RiskScore = memo(() => {
 
   let alertRiskScore: string;
   if (typeof fieldsData === 'string') {
-    alertRiskScore = fieldsData;
+    alertRiskScore = formatRiskScore(Number(fieldsData));
```

Comment:
Not incredibly keen on doing a cast here twice, would be nicer to do in one place but this was a tad more readable than an alternative with ternary operator doing these checks. The second casting is because fieldsData could be a list of strings, although in usage it seems like it never is.